rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -522,6 +553,223 @@ void save_callsFederationSupportSave() throws IOException {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("svp tests")
+    class SvpTests {
+        private Federation activeFederation;
+        private Federation proposedFederation;
+
+        private BtcTransaction svpFundTransactionUnsigned;
+        private Sha256Hash svpFundTransactionHashUnsigned;
+
+        private Transaction rskTx;
+        private Keccak256 rskTxHash;
+
+        private PegoutsWaitingForConfirmations pegoutsWaitingForConfirmations;
+
+        @BeforeEach
+        void setUp() {
+            rskExecutionBlock = mock(Block.class);
+            long rskExecutionBlockNumber = 1000L;
+            when(rskExecutionBlock.getNumber()).thenReturn(rskExecutionBlockNumber);
+
+            activeFederation = FederationTestUtils.getGenesisFederation(federationConstantsMainnet);
```

Comment:
Not sure using the genesis federation is a good idea, it's a legacy standard federation. I'd use the current federation type